### ISSUE TITLE
crashes with Error: listen EADDRINUSE 

### ISSUE THREAD

**Issue by berdof** [ctrlplusb/react-universally#96]
after few minutes of working the app with `npm run development` crashes with this error:

`Error: listen EADDRINUSE :::5100
    at Object.exports._errnoException (util.js:1012:11)
    at exports._exceptionWithHostPort (util.js:1035:20)
    at Server._listen2 (net.js:1252:14)
    at listen (net.js:1288:10)
    at Server.listen (net.js:1384:5)
    at EventEmitter.listen (/Users/berdof/Projects/mine/jnetim/cabinetSimple/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/berdof/Projects/mine/jnetim/cabinetSimple/build/server/webpack:/src/server/index.js:54:1)
    at __webpack_require__ (/Users/berdof/Projects/mine/jnetim/cabinetSimple/build/server/webpack:/webpack/bootstrap b7d04c7a912c94671474:19:1)
    at Object.<anonymous> (/Users/berdof/Projects/mine/jnetim/cabinetSimple/build/server/main.js:30351:18)
    at __webpack_require__ (/Users/berdof/Projects/mine/jnetim/cabinetSimple/build/server/webpack:/webpack/bootstrap b7d04c7a912c94671474:19:1)
`
Only this app is working on following port. I tried to change port but it didn't help. 

Any help? 
Great boilerplate btw:)

**Comment by ctrlplusb** [comment]
Hi @berdof 

Glad you like it! :)

I am sorry to hear you are having problems.  This used to be an old issue, but I haven't been experiencing it for some time now due to the way I changed my dev server script.

Reading through your post I may be misinterpreting a few things, can you confirm the following.  If you change the config so that the app runs on a different port does it not work immediately, or only after a little bit of usage?

**Comment by berdof** [comment]
@ctrlplusb thanks for the fast response. Really appreciate it. 
App sometimes crashes after several hot updates but sometimes it lives for an hour. I have a big project with about 100 components. One incremental build takes about 10sec.  
Can you please share your new dev script?:)

**Comment by ctrlplusb** [comment]
No problem.  

Hmm, I meant the latest version of the dev script. :D

When did you clone/fork the repo?  Are you sure you are running with the latest dev tools stack?

**Comment by berdof** [comment]
Hi! I am using the latest code. Do you know which way I can collect verbose logs when app crashes?

**Comment by ctrlplusb** [comment]
The client and server code run in two seperate express instances, on two different ports, so the first point of call would be to establish which port is in use, and if it maps to the client bundle or the server bundle.

If it is the server bundle listener that throws, you will get an error at this point:
https://github.com/ctrlplusb/react-universally/blob/master/tools/development/server.js#L99

**Comment by ctrlplusb** [comment]
Issue #98 may provide more stability around this.  I am going to see if I can dramatically reduce the complexity of my development server configuration.

**Comment by ctrlplusb** [comment]
See #109, closing in favour of that.
